src/events.ts

```diff
@@ -1,7 +1,7 @@
 import { EventsApi as FSEventsApi, EventsBatchImportApi as FSBatchEventsApi } from '@api/index';
 import { CreateBatchEventsImportJobRequest, CreateBatchEventsImportJobResponse, CreateEventsRequest, CreateEventsResponse, FailedEventsImport, GetBatchEventsImportErrorsResponse, GetBatchEventsImportsResponse, JobStatusResponse } from '@model/index';
 
-import { BatchJob, BatchJobOptions, IBatchRequester } from './batch';
+import { BatchJob, BatchJobOptions, IBatchJob, IBatchRequester } from './batch';
```

Comment:
Just curious: why are some interfaces prepended with an `I` (like `IBatchJob`) and others not (like `BatchJobOptions`).

@sabrina-li ?

Reply:
urgh that is probably an oversight. `BatchJobOptions` was initially a `type` not `interface`
I've created a jira card for cleaning things up. I'm leaning not prefix with `I` at all, seems more typical for TS, open to suggestions tho

Reply:
Yeah, I think match up with TS naming conventions as much as possible. In any case: aim for consistency.